modules/SofaGeneralExplicitOdeSolver/CMakeLists.txt

```diff
@@ -12,22 +12,15 @@ set(SOURCE_FILES
     ${SOFAGENERALEXPLICITODESOLVER_SRC}/initSofaGeneralExplicitOdeSolver.cpp
     )
 
-list(APPEND HEADER_FILES
-    ${SOFAGENERALEXPLICITODESOLVER_SRC}/CentralDifferenceSolver.h
-    ${SOFAGENERALEXPLICITODESOLVER_SRC}/RungeKutta2Solver.h
-    ${SOFAGENERALEXPLICITODESOLVER_SRC}/RungeKutta4Solver.h
-    )
-list(APPEND SOURCE_FILES
-    ${SOFAGENERALEXPLICITODESOLVER_SRC}/CentralDifferenceSolver.cpp
-    ${SOFAGENERALEXPLICITODESOLVER_SRC}/RungeKutta2Solver.cpp
-    ${SOFAGENERALEXPLICITODESOLVER_SRC}/RungeKutta4Solver.cpp
-    )
-
 sofa_find_package(SofaFramework REQUIRED) # SofaSimulationCore
 
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES})
 target_link_libraries(${PROJECT_NAME} PUBLIC SofaCore SofaSimulationCore)
 
+message(WARNING "${PROJECT_NAME} module is deprecated, its modules is in Sofa.Component.ODESolver.Forward.")
```

Comment:
To be done in *Config.cmake.in, like we did with SofaGeneral and other meta-packages.